packages/create-plugin/templates/github/workflows/ci.yml

```diff
@@ -68,14 +67,14 @@ jobs:
 
       - name: Test backend
         if: steps.check-for-backend.outputs.has-backend == 'true'
-        uses: magefile/mage-action@v3
+        uses: magefile/mage-action@6f50bbb8ea47d56e62dee92392788acbc8192d0b # v3.1.0
```

Comment:
are these doomed to be outdated? (I guess no automatic process will update these templates)

Reply:
If the developer set up dependabot they'll be updated.